MediaBrowser.Api/UserLibrary/BaseItemsRequest.cs

```diff
@@ -42,12 +42,6 @@ protected BaseItemsRequest()
         [ApiMember(Name = "MinIndexNumber", Description = "Optional filter by minimum index number.", IsRequired = false, DataType = "int", ParameterType = "query", Verb = "GET")]
         public int? MinIndexNumber { get; set; }
 
-        [ApiMember(Name = "MinPlayers", Description = "Optional filter by minimum number of game players.", IsRequired = false, DataType = "int", ParameterType = "query", Verb = "GET")]
```

Comment:
What bugs me here is this is backwards-incompatible change which, if we strictly follow semver, should make us change version to `11.0.0`... maybe better add some stubs returning some dummy values? cc @joshuaboniface

Reply:
I get what you're saying, but I think it was previously discussed that we will have breaking changes here and there, but once we reach a stable API we should release 11.0.0 (or higher).

I prefer to break shit early and not mock anything unnecessarily. It's just tech debt that no one remembers later.

Reply:
Okay, let's leave it like that for now.
I doubt any client except Web UI ever cared about games anyway :)